uniffi_bindgen/src/lib.rs

```diff
@@ -264,16 +247,11 @@ pub fn generate_external_bindings(
 // such as the `extern "C"` function definitions and record data types.
 pub fn generate_component_scaffolding(
     udl_file: &Utf8Path,
-    config_file_override: Option<&Utf8Path>,
+    _config_file_override: Option<&Utf8Path>,
```

Comment:
My thinking was that it would be useful if we ever did want to load the config file, for example if we added support for a `scaffolding` section.  But that feature has never been used and it's not clear how it would work with proc-macros anyways.  Let's remove it all.